examples/knative/powershell/kn-ps-webhook/handler.ps1

```diff
@@ -0,0 +1,123 @@
+Function Process-Init {
+   [CmdletBinding()]
+   param()
+   Write-Host "$(Get-Date) - Processing Init`n"
+
+   Write-Host "$(Get-Date) - Init Processing Completed`n"
+}
+
+Function Process-Shutdown {
+   [CmdletBinding()]
+   param()
+   Write-Host "$(Get-Date) - Processing Shutdown`n"
+
+   Write-Host "$(Get-Date) - Shutdown Processing Completed`n"
+}
+
+Function Process-Handler {
+   [CmdletBinding()]
+   param(
+      [Parameter(Position=0,Mandatory=$true)]$Headers,
+      [Parameter(Position=1,Mandatory=$true)]$Context
+   )
+
+   Import-Module CloudEvents.Sdk
+
+   Write-host "$(Get-Date) - Processing Context ...`n"
+   try {
+      if ($Context -is [string]) {
+         $Context = [System.Text.Encoding]::UTF8.GetBytes($Context)
+      }
+   } catch {
+      throw "`nFailed to encode `$Context input"
```

Comment:
This should be a more meaningful error message and also documented in the README as which input/data encoding is supported.